src/root/components/navbar.js

```diff
@@ -46,6 +47,18 @@ function Navbar(props) {
   const loadOptionsWithDebouncing = React.useCallback((input, callback) => {
     window.clearTimeout(debounceTimeoutRef.current);
 
+    if (input.length == 2) {
```

Comment:
@frozenhelium @batpad could you please suggest me another condition (instead of input.length == 2) that would be true when the next request is not fulfilled?